src/rpc/rawtransaction.cpp

```diff
@@ -992,6 +992,86 @@ UniValue sendrawtransaction(const JSONRPCRequest& request)
     return hashTx.GetHex();
 }
 
+UniValue testmempoolaccept(const JSONRPCRequest& request)
+{
+    if (request.fHelp || request.params.size() < 1 || request.params.size() > 2) {
+        throw std::runtime_error(
+            // clang-format off
+            "testmempoolaccept \"hexstring\" ( allowhighfees )\n"
+            "\nReturns if raw transaction (serialized, hex-encoded) would be accepted by mempool.\n"
+            "\nAlso see sendrawtransaction call.\n"
+            "\nArguments:\n"
+            "1. \"hexstring\"      (string, required) The hex string of the raw transaction)\n"
+            "2. allowhighfees    (boolean, optional, default=false) Allow high fees\n"
+            "\nResult:\n"
+            "{\n"
+            "  \"hex\"            (string) The transaction hash in hex\n"
```

Comment:
Seems somewhat redundant to return the parameter as-is, no?

Reply:
Thanks, will rename this to `txid` and the parameter to `rawtx` to make clear that they are different.